packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -188,6 +190,7 @@ export function publish(options: InitOptions): void {
       autoLoggingID: getOrSetAutoLoggingID(element),
       flowlet,
       isTrusted,
+      surface,
```

Comment:
We just had another PR adding this. I'll approve and merge that to encourage more people to help! :)